### PR TITLE
feat(ui): Implement new profile pages

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Implement new profile pages

## Motivation and Context:

Address #2317

## How Has This Been Tested?

Manually

## Types of changes:

UI changes

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: renderer/components/Home/WalletsMenu.js [hunk 1]
```diff
@@ -1,47 +1,63 @@
...
+            setActiveWallet(wallet.id)
+            history.push(`/home/wallet/${wallet.id}`)
+          },
+        }
+      })
+
+    return (
+      <Box p={2} {...rest}>
+        <WalletGroupHeader>{title}</WalletGroupHeader>
+        <Menu items={generateMenuItems(wallets)} selectedItem={generateMenuItemId(activeWallet)} />
```
Comment (korhaliv) [inline renderer/components/Home/WalletsMenu.js]: Not sure if it's a problem, but `activeWallet` can be undefined, so `generateMenuItemId` will also generate something weird

File: renderer/components/UI/Menu.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box } from 'rebass'
+import MenuItem from 'components/UI/MenuItem'
+
+const Menu = ({ items, selectedItem, ...rest }) => {
+  return (
+    <Box {...rest}>
+      {items.map(({ id, title, onClick }) => {
+        const handleClick = e => onClick(e.target.value)
```
[the remark this answers is not in the document]
Comment (mrfelton) [reply renderer/components/UI/Menu.js]: Updated

File: renderer/components/Profile/ProfilePaneConnect/ProfilePaneConnect.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                  <FormattedMessage {...messages[buttonMessage]} />
+                </Button>
+                <Message justifyContent="center" my={3} variant="warning">
+                  <FormattedMessage {...messages.lndconnect_warning} />
+                </Message>
+              </Box>
+            }
+            py={2}
+            right={
+              <>
```
[the remark this answers is not in the document]
Comment (mrfelton) [reply renderer/components/Profile/ProfilePaneConnect/ProfilePaneConnect.js]: Updated

File: renderer/components/Profile/ProfilePaneNodeInfo/ProfilePaneNodeInfo.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            hint={intl.formatMessage({ ...messages.copy_pubkey })}
+            my={3}
+            onCopy={notifyOfCopy}
+            value={identityPubkey}
+          />
+        </>
+      )}
+      <DataRow
+        left={
+          <>
```
[the remark this answers is not in the document]
Comment (mrfelton) [reply renderer/components/Profile/ProfilePaneNodeInfo/ProfilePaneNodeInfo.js]: Updated

**Comment by mrfelton** [comment]
Have made those updates @korhaliv and refactored the new info selectors based on  our slack convo

**Pull request merged**